README.md

```diff
@@ -20,27 +20,18 @@
 
 
 ##### Code
-
+- Grab the das-apply-service configuration json file from [das-employer-config](https://github.com/SkillsFundingAgency/das-employer-config/blob/master/das-roatp-oversight/SFA.DAS.RoatpOversight.json)
 - Create a Configuration table in your (Development) local Azure Storage account.
-- Add a row to the Configuration table with fields: PartitionKey: LOCAL, RowKey: SFA.DAS.RoatpOversight_1.0, Data: {
-	"SessionRedisConnectionString": "localhost",
-	"StaffAuthentication": {
-		"WtRealm": "https://localhost:45667",
-		"MetadataAddress": "https://adfs.preprod.skillsfunding.service.gov.uk/FederationMetadata/2007-06/FederationMetadata.xml"
-	},
-	"RoatpApplicationApiAuthentication": {
-		"TenantId": "tenantId Guid",
-		"ClientId": "clientId Guid",
-		"ClientSecret": "hidden password",
-		"ResourceId": "url to resource id",
-		"ApiBaseAddress": "https://localhost:6000"
-	}
-}
-
-Note for the RoatpApplicationApiAuthentication, you can copy the settings for TenantId, ClientId, ClientSecret, ResourceId from the das-apply-service configuration json file from [das-employer-config](https://github.com/SkillsFundingAgency/das-employer-config/blob/master/das-apply-service/SFA.DAS.ApplyService.json) using the settings under section 'AssessorServiceApiAuthentication'
+- Add a row to the Configuration table with fields: PartitionKey: LOCAL, RowKey: SFA.DAS.RoatpOversight_1.0, Data: {The contents of the local config json file}.
+- Alter the SqlConnectionString value in the json to point to your database.
 
 ### Running the code
 
 - `dotnet run` the following projects:
   - SFA.DAS.RoatpOversight.Web
 - Navigate to (https://localhost:45667) and you should see the start page.
+
+- run the following repos locally to get the application working, and dashboard links working suitably
+  - applyService
```

Comment:
Can these be links to the repos?
Need das-roatp-service in there too

Reply:
good idea